aiohttp/cookiejar.py

```diff
@@ -175,8 +189,9 @@ def update_cookies(self,
                 if expires:
                     expire_time = self._parse_date(expires)
                     if expire_time:
-                        self._expire_cookie(expire_time.timestamp(),
-                                            domain, name)
+                        self._expire_cookie(datetime.datetime.fromtimestamp(
+                            expire_time.timestamp(), tz=datetime.timezone.utc),
+                            domain, name)
```

Comment:
The `expire_time` value here is a perfectly serviceable UTC value already, why convert again?